Submit_nomination

```diff
@@ -0,0 +1,156 @@
+Public Function Submit_nomination()
+
+ 		Set DataBuilder = New DataClass
+			
+		sWorkBook = Path        'WorkBook
+		vSheet = Sheet                'WorkSheet
+		iRow = Row                        'Row Number
+		
+		Set Data = DataBuilder.Load(sWorkBook, vSheet, iRow)
+ Wait(10)
+
+'Parameters required to submit nomination for the nominee(s)
+promoName = data("Promotion_Name")'"GM-Nomination"
+nomineeType = data("Team")'"Individual"
+nomineeBehavior = "Ideas"
+nomineeCard = "1st Year Celebration"
+nomineeComment = "Bob is the hardest working person that I know. he is the first to arrive and the last to leave." & _
+				 "he is always positive and provides a pleasant working environment. He always greets you with a smile. Bob is the rock of the QA services" & _
+				 "he is what makes us great! Bob’s outstanding performance is a daily occurrence. Examples of his outstanding performance can be witnessed" &_
+				 "and appreciated whenever you have an interaction with him. Everyday he provides a can do attitude that is contagious to everybody."
+
+searchBy = "Last Name"
```

Comment:
Here my Comments are:
1. searchBy field should be dynamic and add more variables
2. Remove Wait functions
Defect has been raised the same. Once you fix the code make the update in issue